app/src/main/java/io/intrepid/skeleton/logging/TimberLogger.java

```diff
@@ -0,0 +1,41 @@
+package io.intrepid.skeleton.logging;
+
+import android.support.annotation.NonNull;
+import android.util.Log;
+
+import io.intrepid.skeleton.BuildConfig;
+import timber.log.Timber;
+
+public class TimberLogger {
```

Comment:
How about `TimberConfig`?  This sort of implies to me that this class itself is the logger.